README.md

```diff
@@ -542,6 +543,14 @@ const results = await collection.getSearch().dense({
 
 <br>
 
+# 🙌 Contributing
+
+We welcome contributions from the community! Whether it's _fixing a bug_, _improving documentation_, or building _new features_—every bit helps.
+
+Please read our [CONTRIBUTING.md](CONTRIBUTING.md) guide to get started.
```

Comment:
For full guidelines (coding standards, commit messages, CI checks), please see our [CONTRIBUTING.md](CONTRIBUTING.md). If you have any questions, feel free to open an issue or join the discussion on [Discord](https://discord.gg/WbSbXYWvta). We can’t wait to collaborate with you!